performance/front/table-and-chart-with-good-performance.md

```diff
@@ -0,0 +1,41 @@
+# A table or a chart on my project takes to much time to render
```

Comment:
As it enters in : `Now that you investigated, can we help you?` I would call it : `Optimize the render time of lists, tables and charts`

Isn't it a Method of Operation ? `[MO] Optimize the render time of lists, tables and charts`

Little english mistake : `too much time`